samples/nrf5340/multicore/README.rst

```diff
@@ -0,0 +1,64 @@
+.. _nrf5340_multicore:
+
+nRF5340: Multicore application
+###############################
+
+.. contents::
+   :local:
+   :depth: 2
+
+You can use this sample to see how to build an application on the network core as a child image from the sample source files.
+
+Requirements
+************
+
+The sample supports the following development kits:
+
+.. table-from-rows:: /includes/sample_board_rows.txt
+   :header: heading
+   :rows: nrf5340dk_nrf5340_cpuapp
+
+Overview
+********
+
+The sample demonstrates how to build a multicore application with the :c:macro:`add_child_image` CMake macro.
+For general information about multi-image builds, see :ref:`ug_multi_image`.
+When building any multi-image application, the build system in the |NCS| adds the child images based on the options selected for the parent image.
+This sample shows how to inform the build system about dedicated sources for the child image (in the :file:`zephyr` and :file:`aci` directories).
```

Comment:
```suggestion
If you use the :c:macro:`add_child_image` macro, this sample informs the build system about dedicated sources for the child image (in the :file:`zephyr` and :file:`aci` directories).
```
Added this, as the issues is only when the user uses `add_child_image` macro. Hope this didn't change the meaning.

Reply:
It cant be like that. I will keep the old version.